features/aanhef.feature

```diff
@@ -0,0 +1,80 @@
+# language: nl
+
+Functionaliteit: Als gemeente wil ik de juiste en consistente briefaanhef in communicatie naar burgers
+  Attribuut "aanhef" bij een natuurlijk persoon wordt gevuld om de persoon op eenduidige wijze te kunnen aanschrijven.
+  De aanhef wordt gebruikt bovenaan een brief.
+
+  Attribuut aanschrijfwijze wordt samengesteld op basis van:
+  - adellijkeTitelOfPredikaat
+  - aanduidingNaamgebruik
+  - naam
+    - geslachtsnaam
+    - voornamen
+    - voorvoegselsgeslachtsnaam
+  - geslacht
+    - geslachtsaanduiding
+  - heeftPartnerschap
+    - naamPartner
+      - geslachtsnaam
+      - voorvoegselsgeslachtsnaam
+
+  In de BRK is niet bekend of de partner een adellijke titel heeft. In de aanhef wordt daarom nooit een adellijke titel van de partner gebruikt.
+
+  In onderstaande tabellen betekenen de afkortingen voor naamcomponenten:
+  - AH = aanhef afhankelijk van geslachtsaanduiding:
+    - Als geslachtsaanduiding==1 (Man) Dan “Geachte heer”
+    - Als geslachtsaanduiding==2 (Vrouw) Dan “Geachte mevrouw"
+    - Als geslachtsaanduiding==3 (Onbekend) Dan “Geachte heer/mevrouw"
+  - VV = naam.voorvoegselsgeslachtsnaam
+  - GN = naam.geslachtsnaam
+  - VP = heeftPartnerschap.naamPartner.voorvoegselsgeslachtsnaam
+  - GP = heeftPartnerschap.naamPartner.geslachtsnaam
+
+  Samenstelling van aanhef voor een persoon zonder adellijke titel of predikaat:
+  | aanduidingNaamgebruik | aanschrijfwijze |
+  | E                     | AH VV GN        |
+  | N                     | AH VV GN-VP GP  |
+  | P                     | AH VP GP        |
+  | V                     | AH VP GP-VV GN  |
```

Comment:
Het is mij onduidelijk waar de `E`, `N`, `P`, `V` voor staan. Kan dit ergens worden toegelicht? Geldt ook voor de andere features waar dit gebruikt wordt.

Reply:
Dit zijn codes uit de volgende IMKAD waardenlijst: https://www.kadaster.nl/schemas/waardelijsten/BRPAanduidingNaamgebruik/. 
Misschien hier naar toe verwijzen?

Reply:
ik heb de betekenissen van de codes opgenomen bij de lijst afkortingen